substrafl/remote/register/register.py

```diff
@@ -27,7 +27,7 @@
 logger = logging.getLogger(__name__)
 
 # Substra tools version for which the image naming scheme changed
-MINIMAL_DOCKER_SUBSTRATOOLS_VERSION = "0.10.0"
+MINIMAL_DOCKER_SUBSTRATOOLS_VERSION = "0.16.0"
```

Comment:
do not change this line, the `MINIMAL_DOCKER_SUBSTRATOOLS_VERSION` is relative to the naming scheme

Reply:
but the previous images do not exist on ghcr.io/substra/substra-tools ?

Reply:
ah true, should have been changed when we changed the image name